sockets/socket_io.py

```diff
@@ -31,6 +32,11 @@ def get_sid(url, headers, cookies, origin):
     if origin:
         custom_headers['Origin'] = origin
 
+    if no_cert_check:
+        # disable warning caused by disabled certificate verification
+        import urllib3
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Instead of importing urllib3, it can be done with requests library too which is already imported.
Replace it with the below code and check if it is working for your test cases.
`requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)`

Reply:
Thanks for the hint!